app/views/HomeView.scala.html

```diff
@@ -38,20 +37,38 @@
     </p>
 }
 
+    @alertBannerHtml = {
+        @viewModel.showAlertBanner.map { alert =>
+            @viewModel.verifyOrBouncedUrl.map { url =>
+                @if(alert.isInstanceOf[PaperlessStatusUnverified]) {
+                    <p class="govuk-notification-banner__heading">@messages("alert_banner.alert_unverified_email.p1") <a href=@url>@messages("alert_banner.alert_unverified_email.link")</a>.</p>
+                }
+                @if(alert.isInstanceOf[PaperlessStatusBounced]) {
+                    <p class="govuk-notification-banner__heading">@messages("alert_banner.alert_bounced_email.p1") <a href=@url>@messages("alert_banner.alert_bounced_email.link")</a>
+                        @messages("alert_banner.alert_bounced_email.p2")</p>
+                }
+            }
```

Comment:
In the future that could become a long list. Could we pass a List[Html] or similar? For now it will be only one element.